### DISCUSSION TITLE
Using hover and pointer media queries

### DISCUSSION THREAD

**Discussion by Ericlm** [tailwindlabs/tailwindcss#11423] [Help]
Hello there! I was wondering how to generate custom media queries, especially for the [hover](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/hover) and [pointer](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/pointer) queries ; any help would be appreciated 🙂

**Comment by wongjn** [comment]
You could either:

- [Use the `screens` configuration](https://tailwindcss.com/docs/screens#custom-media-queries).
- [Use a Tailwind plugin to register new variants](https://tailwindcss.com/docs/plugins#static-variants).
- [Use an arbitrary variant](https://tailwindcss.com/docs/hover-focus-and-other-states#using-arbitrary-variants).

**Reply by sghong3130** [reply]
thanks
